lib/msal-browser/test/app/PublicClientApplication.spec.ts

```diff
@@ -3888,37 +3858,29 @@ describe("PublicClientApplication.ts Class Unit Tests", () => {
                 tokenType: AuthenticationScheme.BEARER,
             };
 
-            const nativeAcquireTokenSpy = sinon
-                .stub(NativeInteractionClient.prototype, "acquireToken")
-                .callsFake(async () => {
-                    throw new NativeAuthError(
-                        "ContentError",
-                        "error in extension"
-                    );
-                });
-            const silentSpy = sinon
-                .stub(SilentIframeClient.prototype, "acquireToken")
-                .callsFake(async () => {
-                    return testTokenResponse;
-                });
+            const nativeAcquireTokenSpy: jest.SpyInstance = jest
+                .spyOn(NativeInteractionClient.prototype, "acquireToken")
+                .mockRejectedValue(
+                    new NativeAuthError("ContentError", "error in extension")
+                );
+            const silentSpy: jest.SpyInstance = jest
+                .spyOn(SilentIframeClient.prototype, "acquireToken")
+                .mockResolvedValue(testTokenResponse);
             const silentRequest = {
                 scopes: ["User.Read"],
                 account: testAccount,
             };
             const response = await pca.acquireTokenSilent(silentRequest);
 
             expect(response).toEqual(testTokenResponse);
-            expect(nativeAcquireTokenSpy.calledOnce).toBeTruthy();
-            expect(silentSpy.calledOnce).toBeTruthy();
-            expect(
-                silentSpy.calledWith({
-                    ...silentRequest,
-                    tokenQueryParameters: {
-                        "x-client-current-telemetry":
-                            "||broker_error=ContentError",
-                    },
-                })
-            );
+            expect(nativeAcquireTokenSpy).toHaveBeenCalledTimes(1);
+            expect(silentSpy).toHaveBeenCalledTimes(1);
+            expect(silentSpy).toHaveBeenCalledWith({
+                ...silentRequest,
+                tokenQueryParameters: {
```

Comment:
This tokenQueryParameters object doesn't exist in the spy's parameters

Reply:
I'm not sure why this was provided before or how the test worked before, but looking through the code we don't ever set tokenQueryParameters, can be removed.